src/json-editor.component.scss

```diff
@@ -128,8 +131,8 @@ td.button-holder, th.button-holder {
   width: 46px;
 }
 
-div.error {
-  background: #f2968d;
+td.error {
+  background-color: #f2968d !important;
```

Comment:
is important needed? There is more than one `error` class inside the `td`? We can also give the class a more meaningful name to avoid adding important.